public/configs/otherMarketData.json

```diff
@@ -2060,15 +1919,15 @@
       "amount": "10000000000000000000000"
     },
     "meta": {
-      "assetName": "Flow - Dapper Labs",
-      "referencePrice": 1.1556986926618635
+      "assetName": "Flow",
+      "referencePrice": 0.9147591525078125
     }
   },
   "FLR": {
     "title": "Add FLR-USD perpetual market",
     "summary": "Add the x/prices, x/perpetuals and x/clob parameters needed for a FLR-USD perpetual market. Create the market in INITIALIZING status and transition it to ACTIVE status after 3600 blocks. Added via the new market widget.",
     "params": {
-      "id": 38,
+      "id": 316,
```

Comment:
[q] why do the ids change?

Reply:
they should increment by 1, will close this pr and open a new one with the fix!